client-src/elements/form-field-specs.js

```diff
@@ -366,6 +371,24 @@ export const ALL_FIELDS = {
         public forum before sending an Intent to Prototype in
         order to enable discussion with other browser vendors,
         standards bodies, or other interested parties.`,
+    extra_help: html`
+        <p>
+        See the TAG guide to writing <a href="https://tag.w3.org/explainers/">Explainers</a>
+        for several examples of good explainers and tips for effective explainers.
+        </p>
+        <p>
+        You will need to publish this explainer by sharing it on a public forum,
+        and kick off "standards incubation" by proposing this to a relevant standards venue.
+        If you've already made an initial public proposal, post your explainer to that thread. Otherwise, your initial proposal can refer to your explainer.
+        </p>
+        <p>
+        Once a second organization is interested in the WICG proposal, you can move the explainer into the WICG. The <a href="https://wicg.github.io/admin/charter.html#chairs">WICG co-chairs</a> can help you.
+        </p>
```

Comment:
I think this doesn't really belong here. We ought to have a field that captures the plans around moving to a CG, and this help text would be useful there, but until that field exists, I'm inclined to leave this paragraph out.
```suggestion
```

Reply:
done